src/program/snabbnfv/nfvconfig.lua

```diff
@@ -23,27 +23,29 @@ function load (file, pciaddr, sockpath)
       print(format("could not find device information for PCI address %s", pciaddr))
       main.exit(1)
    end
+   local driver = require(device_info.driver).driver
 
    local ports = lib.load_conf(file)
    local c = config.new()
    for _,t in ipairs(ports) do
-      local vlan, mac_address = t.vlan, t.mac_address
+      local vlan, mac_address, vmdq = t.vlan, t.mac_address, t.vmdq
+      local disable_mrg_rxbuf = t.disable_mergeable_rx_buffer
       local name = port_name(t)
       local NIC = name.."_NIC"
       local Virtio = name.."_Virtio"
-      local vmdq = true
+      if vmdq == nil then vmdq = true end
```

Comment:
I'm not sure about this change. It's necessary that vmdq is false. Right now in master vmdq is only false if `mac_address` is not set in a `ports.cfg` file. Would it be possible to set a `mac_address` and disable `vmdq`? cc @lukego